devops/eslint-config/index.js

```diff
@@ -43,6 +43,36 @@ module.exports = {
     // should prefer using 'debug' package at least to allow control of
     // output verbosity if logging to console.
     'no-console': 'off',
+    '@typescript-eslint/camelcase': 'off',
+    '@typescript-eslint/class-name-casing': 'off',
+    "@typescript-eslint/naming-convention": [
+      "error",
+      {
+        selector: 'default',
+        format: ['camelCase'],
```

Comment:
I'm trying to understand the naming-conventions rules:
Is the format array listing what is allowed or considered and error?

Reply:
Those are allowed